jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/resources/META-INF/CaseMgmtorm.xml

```diff
@@ -598,10 +646,12 @@
         log.processInstanceId, <!-- case top level process instance id -->
         log.outcome,
         log.slaDueDate,
-        log.slaCompliance
+        log.slaCompliance,
+        parent.correlationKey
       )
       from
         ProcessInstanceLog log
+      left join ProcessInstanceLog parent on log.parentProcessInstanceId = parent.processInstanceId
```

Comment:
I am bit concerned with adding this extra join to all queries, again... we need to assess the impact on performance

Reply:
above